src/app/backend/handler/apihandler.go

```diff
@@ -735,15 +570,21 @@ func (apiHandler *APIHandler) handleGetRbacRoleBindingList(request *restful.Requ
 
 func (apiHandler *APIHandler) handleGetCsrfToken(request *restful.Request, response *restful.Response) {
 	action := request.PathParameter("action")
-	token := xsrftoken.Generate(apiHandler.csrfKey, "none", action)
-	response.WriteHeaderAndEntity(http.StatusOK, CsrfToken{Token: token})
+	token := xsrftoken.Generate(apiHandler.manager.CSRFKey(), "none", action)
+	response.WriteHeaderAndEntity(http.StatusOK, api.CsrfToken{Token: token})
 }
 
 func (apiHandler *APIHandler) handleGetStatefulSetList(request *restful.Request, response *restful.Response) {
+	k8sClient, err := apiHandler.manager.Client(request)
+	if err != nil {
```

Comment:
We can't really handle errors while getting client because it is created from scratch on every request and we could end up in returning nil here without error, and that would lead to panic error. We can't also panic when client is not created. We can however test these methods eventually similar to how I have tested client manager.